_data/doc_categories.yml

```diff
@@ -0,0 +1,12 @@
+# The order of the categories in this file (first-to-last) determines their
+# order in the UI (left-to-right)
+- title: Overview
+  href: docs
```

Comment:
I'm not sure what href: is here, but could permalink front-matter help?

https://jekyllrb.com/docs/permalinks/